source/_integrations/weatherflow.markdown

```diff
@@ -0,0 +1,65 @@
+---
+title: "WeatherFlow"
+description: "WeatherFlow weather sensor"
+ha_release: "0.38"
+ha_category: 
+  - Sensor
+ha_platforms:
+ - sensor
+ha_iot_class: "Local Polling"
+ha_quality_scale: silver
+ha_config_flow: true
+ha_codeowners:
+  - '@natekspencer'
+  - '@jeeftor'
+ha_domain: weatherflow
+---
+
+
+### WeatherFlow 
+
+The WeatherFlow integration uses the local UDP interface on a Weatherflow sensor, such as the Tempest, to discover and display weather data gathered vial the local network.
+
+### Installation Notes
+
+When you add this integration it may take a few minutes for all the entities to show up. The integration will listen for WeatherFlow UDP packets over the network and decode them as they arrive. As sensor-specific packets show up the sensor entity will be created.
+
+### Minor differences in calculated readings
+
+WeatherFlow sensors provide direct data for some sensors, where as other sensor values are the results of calculations. This integration utilizes [PsychroLib](https://github.com/psychrometrics/psychrolib) library to generate these calculations and as such there may be some drift from what is displayed in the Tempest Mobile App.
```

Comment:
- Integration should not mess with data, that is not allowed. The fact that this comment is needed, shows the integration code is violating general standards we use in Home Assistant
- Linking the library isn't needed, as it is not of use to the end-user. We should not send our users into Python code.